app/assets/javascripts/discourse/app/lib/keyboard-shortcuts.js

```diff
@@ -124,7 +124,15 @@ export default {
     this.container = null;
   },
 
+  isTornDown() {
```

Comment:
this name doesn’t strike me as very explicit, I have no idea what it does just reading it

Reply:
I mean, it just means that `teardown`, the function above it, has been run. If you try and bind shortcuts after this has been done you get errors.